ajax/libs/kule.lazy/package.json

```diff
@@ -20,7 +20,7 @@
     "responsive",
     "browsers"
   ],
-  "filename": "css/kule-lazy-full.min.css",
+  "filename": "dist/css/kule-lazy-full.min.css",
```

Comment:
i tried some methods, include:
`"filename": "kule-lazy-full.min.css",`
`"filename": "css/kule-lazy-full.min.css",`

but both has error
kule.lazy 
      ✗ kule.lazy: filename from package.json exists 
        » ./ajax/libs/kule.lazy/3.0.1101beta/css/kule-lazy-full.min.css does not exist but is referenced in package.json! // valid-packages-test.js:101 
  ✗ Broken » 21876 honored ∙ 1 broken (5.554s) 
  npm ERR! Test failed.  See above for more details.

npm test returned exit code 1

so...i think that need to add `dist/`
It will success